app/src/main/java/com/example/movieapp/viewModel/MainViewModel.kt

```diff
@@ -28,24 +28,28 @@ class MainViewModel : ViewModel() {
     /**
      * Type of the list displayed
      */
-    var TypeDisplay: TypeDisplay = com.example.movieapp.viewModel.TypeDisplay.POPULAR
+    var typeDisplay: TypeDisplay = com.example.movieapp.viewModel.TypeDisplay.POPULAR
 
     /*
     return the list of elements necessary
      */
     fun getListCurrent(): MutableLiveData<List<Movie>> {
-        return when (TypeDisplay) {
+        return getList(typeDisplay)
```

Comment:
always return a LiveData not a MutableLiveData

Reply:
done